pkg/app/login.go

```diff
@@ -151,5 +151,5 @@ func generateLoginName() (string, error) {
 		return "", errors.Wrap(err, "failed to get the user")
 	}
 
-	return fmt.Sprintf("%s@%s", username.Username, hostname), nil
+	return fmt.Sprintf("%s-%s", username.Username, hostname), nil
```

Comment:
What happens to @?

Reply:
Kubernetes does not allow @ in the label.